app/models/service.rb

```diff
@@ -203,6 +204,10 @@ def orchestration_stacks
     service_resources.where(:resource_type => 'OrchestrationStack').collect(&:resource)
   end
 
+  def generic_objects
+    service_resources.where(:resource_type => 'GenericObject').includes(:resource).collect(&:resource)
```

Comment:
👍  A separate PR should be created to add `.includes(:resource)` to the `orchestration_stacks` methods above.